src/main.js

```diff
@@ -172,26 +168,27 @@ async function shuttingDown(eventType, err) {
 }
 module.exports.shuttingDown = shuttingDown;
 
-const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
+const commandFiles = fs.readdirSync(path.join(__dirname, 'commands')).filter(file => file.endsWith('.js'));
 for (const file of commandFiles) {
 	/** @type {{data: import('@discordjs/builders').SlashCommandBuilder, checks: string[], permissions: {user: string[], bot: string[]}, execute(interaction: import('discord.js').CommandInteraction): Promise<void>}} */
-	const command = require(`./commands/${file}`);
+	const command = require(path.join(__dirname, 'commands', file));
 	bot.commands.set(command.data.name, command);
 }
 
-const componentsFolders = fs.readdirSync('./components');
+const componentsFolders = fs.readdirSync(path.join(__dirname, 'components'));
 for (const folder of componentsFolders) {
-	const componentFiles = fs.readdirSync(`./components/${folder}`).filter(file => file.endsWith('.js'));
+	const componentFiles = fs.readdirSync(path.join(__dirname, 'components', folder)).filter(file => file.endsWith('.js'));
 	for (const file of componentFiles) {
-		const component = require(`./components/${folder}/${file}`);
+		const component = require(path.join(__dirname, 'components', folder, file));
+		if (!bot[folder]) bot[folder] = new Collection();
```

Comment:
```js
bot.buttons = new Collection();
bot.selects = new Collection();
```
You should either update `interactionCreate.js` accordingly or update the folder names in `src/components` because you are creating the collections based inside the `component folder's folder names, with that `bot.buttons` are now `bot.button` from `src/components/button`. Same for `bot.selects`, now `bot.selectmenu` from `src/components/selectmenu`.

It means buttons and select menus would no longer work.